.github/stale.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
```

Comment:
That's true, but if I think of issues that are opened now, (to me) it feels like an issue without a response for a month feels stale

I would love to hear what other people think - @BasThomas especially (since you handle closing so many of our current stale issues 😋)

Totally fine with 60 otherwise, Im just trying to make sure our bot is useful